logic/src/test/java/org/itsallcode/whiterabbit/logic/autocomplete/AutocompleteServiceTest.java

```diff
@@ -53,6 +55,27 @@ void dayCommentAutocompleter()
         assertThat(entries).extracting(AutocompleteProposal::getMatchLength).containsExactly(4, 4);
     }
 
+    @Test
+    void dayCommentAutocompleterCachesEntries()
+    {
+        simulateDays(dayRecordsWithComments(null, "", "Comment A", "Comment B"));
+        final AutocompleteEntrySupplier completer = autocompleteService.dayCommentAutocompleter();
+        completer.getEntries("comm");
+        completer.getEntries("comm");
+        verify(storageMock, times(1)).getLatestDays(any());
```

Comment:
Ah, I see what you did there. If you named the test something like `testSecondAutocompletedDayRecordGetsServedFromCache` the test would be easier to grasp.